restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= Restcomm Release Notes
```

Comment:
Perhaps "Restcomm Connect Release Notes" ?

Reply:
Changed it